src/components/SummaryPanel/ErrorRatePieChart.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+
+import { PieChart } from 'patternfly-react';
+
+type ErrorRatePieChartPropType = {
+  percentError: number;
```

Comment:
The data in the graph is not going to be percentage, its going to be a fraction "error rate" So the data we put in this component won't be a percentage value. Right @jshaughn ?

Reply:
I think percentError (a percentage) is more useful to show than RateError (requestsPerSec returning failure status codes (> 2XX)).  The % can be calculated either by the caller or here given total rate and failure rate, which are supplied in the graph nodes/edges.

Reply:
see my comment below where the percentError is set for another question.